installscripts/dockerfiles/gitlab/privatetoken.py

```diff
@@ -0,0 +1,65 @@
+#coding: utf-8
+import sys
+import requests
+from urlparse import urljoin
+from bs4 import BeautifulSoup
+
+endpoint = u"http://localhost"
+root_route = urljoin(endpoint, u"/")
+sign_in_route = urljoin(endpoint, u"/users/sign_in")
+pat_route = urljoin(endpoint, u"/profile/personal_access_tokens")
+
+login = u"root"
```

Comment:
Lets use JazzAdmin email address as the login. If email address isn't allowed, then userid based on email address.

Reply:
I need to check about this. But I would suggest we have these as a separate ticket as these are would be an enhancement type.